fiftyone/core/threed/splat.py

```
@@ -0,0 +1,115 @@
+        )
+
+        normalized_format = (
+            None
+            if format is None
+            else str(format).strip().lower().removeprefix(".")
+        )
+        if (
+            normalized_format is not None
+            and normalized_format not in SUPPORTED_GAUSSIAN_SPLAT_FORMATS
+        ):
+            raise ValueError("Unsupported Gaussian splat format '%s'" % format)
```

Ruff (0.15.21)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)